.dockerignore

```diff
@@ -1,13 +1,20 @@
-**/node_modules/
-**/node_modules_linux/
+**/.circleci/
+**/.vscode/
+**/.idea/
+**/coverage/
 **/docs/
-**/Dockerfile
+**/node_modules/
+**/node_modules_linux/g
```

Comment:
Same question here

Reply:
The `g` leter at the end is a typo. About including the `node_modules_linux`, for the first version of the Docker files I have used [CloudNativeJs](https://cloudnativejs.io) templates as I mentioned in my comments before the first pull request. [here](https://github.com/CloudNativeJS/docker/blob/master/.dockerignore) is the template. I didn't use Linux with Nod.js at all, so my assumption was this is similar to node_modules for Linux system.

Reply:
Let's remove it then.  Things we don't need/understand shouldn't go in here, as they are something we can add when we do need them.